packages/iTwinUI-react/src/core/ExpandableBlock/ExpandableBlock.test.tsx

```diff
@@ -5,7 +5,7 @@
 import { fireEvent, render } from '@testing-library/react';
 import React from 'react';
 import { StatusIconMap } from '../utils';
-import { SvgPlaceholder } from '@itwin/itwinui-icons-react';
+import { SvgMore as SvgPlaceholder } from '../utils';
```

Comment:
This import can be combined with the `StatusIconMap` import

Reply:
Done